doc/source/api.rst

```diff
@@ -1,39 +0,0 @@
-
-API
-===
-
-
-
-.. autosummary::
```

Comment:
Are we sure all of these got re-documented someplace else?

Reply:
Reverted to leave this as is.